symphony/lib/toolkit/class.xsrf.php

```diff
@@ -23,7 +23,11 @@ class XSRF
      */
     public static function getSessionToken()
     {
-        $token = $_SESSION[__SYM_COOKIE_PREFIX__]['xsrf-token'];
+        $token = null;
+
+        if (isset($_SESSION[__SYM_COOKIE_PREFIX__]['xsrf-token'])) {
+            $token = $_SESSION[__SYM_COOKIE_PREFIX__]['xsrf-token'];
+        }
```

Comment:
Actually I hoped that some PHP guru knows a more elegant way to do it, without losing explicity. (As mentioned above, I wouldn't want to hide such patterns in custom functions.)